sources/packages/backend/apps/db-migrations/src/sql/Queue/Add-assessment-workflow-queue-retry.sql

```diff
@@ -0,0 +1,14 @@
+INSERT INTO
+    sims.queue_configurations(queue_name, queue_configuration)
+VALUES
+    (
+        'assessment-workflow-queue-retry',
+        '{
+            "cron": "0 0 * * * ?",
+            "retry": 3,
```

Comment:
As the scheduler itself is for retry, do we need retry on this?

Reply:
The scheduler will be executed every hour, in case it fails there is no need to wait another hour for a retry.
We did not use the retry for short scheduled services, for instance, 30s intervals.
IMO we should continue as it was planned in the ticket 😉